react/Components/input.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext, FC } from 'react'
+import { Input } from 'vtex.styleguide'
+
+import Context from '../Context/context'
+
+interface Props {
+  name: string
+}
```

Comment:
Using string in this case your component is vulnerable. Because, if other person use this component setting other word that is not what do you want, your component will broken.

Use `type` type in this case is a better approach 😃 
So, when you use the component, the `name` prop will "limit" your options in `name` or `text`

```suggestion
type inputType = 'name' | 'text'

interface Props {
  name: inputType 
}
```